src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/RadioLabel/index.tsx

```diff
@@ -16,11 +16,22 @@ import {
   StyledListItemIcon,
```

Comment:
Not necessarily relevant for this PR, but if we do work on this again, probably this file should be split up. I feel like it's trying to do too much.

Reply:
Yeah, I agree. Also, I think it's a bit of a stretch for the `label` to carry as much weight as it's doing in the modal radio group. There's some weird scrolling effects that are starting to happen because the extended "label" of a selection is pretty much as tall as the entire modal. The radio group is trying to helpfully move the item into view when clicked, but because the whole item is so tall now, it makes things worse.

Reply:
https://app.shortcut.com/genepi/story/193962/split-up-refactor-create-nextstrain-tree-modal-radiolabel-into-multiple-files